firmware/boron-ins-fsm/clang-format-all.py

```diff
@@ -0,0 +1,13 @@
+#
+# clang-format-all.py
+#
+# Formats all files with filetype .cpp or .h in the src and test directories using clang-format. 
```

Comment:
Can you please add `.ino` to the comment too?

Reply:
Thank you for the reminder